tethysext/atcore/controllers/resource_workflows/results_views/report_workflow_results_view.py

```diff
@@ -0,0 +1,221 @@
+"""
+********************************************************************************
+* Name: report_workflow_results_view.py
+* Author: nswain, htran, msouffront
+* Created On: October 14, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import logging
+from tethysext.atcore.models.resource_workflow_results import ReportWorkflowResult
+from tethysext.atcore.controllers.resource_workflows.map_workflows import MapWorkflowView
+from tethysext.atcore.controllers.resource_workflows.workflow_results_view import WorkflowResultsView
+from tethysext.atcore.models.resource_workflow_results import DatasetWorkflowResult, PlotWorkflowResult,\
+    SpatialWorkflowResult
+from tethysext.atcore.controllers.utilities import get_plot_object_from_result, get_tabular_data_for_previous_steps
+
+from tethys_sdk.gizmos import DataTableView
+from collections import OrderedDict
+
+
+log = logging.getLogger(__name__)
+
+
+class ReportWorkflowResultsView(MapWorkflowView, WorkflowResultsView):
```

Comment:
If you are creating your own Leaflet map for the report view, then You shouldn't need to inherit from MapWorkflowView here. Do you need this?